addons/mail/static/src/components/message_list/message_list.js

```diff
@@ -143,6 +143,7 @@ class MessageList extends Component {
                 case 'change-of-thread-cache':
                 case 'home-menu-hidden':
                 case 'home-menu-shown':
+                case 'change-of-position':
```

Comment:
IMO I dislike (component) hint. We did this to solve scroll issues, but the way we did is by broadcasting dynamic events and adjusting as a result, without any proper (persistent) representation in the global state.

It's a change for a later date, but for the time being, let's try to minimise naming problems with this pattern.
Instead of `change-of-position`, I suggest just naming it `adjust-scroll`. Just ignore other component hint names that are not affected by this PR.